pkg/v1/tkg/managementcomponents/management_component_install.go

```diff
@@ -48,18 +60,182 @@ type ManagementComponentsInstallOptions struct {
 	ManagementPackageRepositoryOptions ManagementPackageRepositoryOptions
 }
 
+func generateAddonSecretName(clusterName, addonName string) string {
+	return fmt.Sprintf("%s-%s-addon", clusterName, fmt.Sprintf("tanzu-%s", addonName))
```

Comment:
Question: Are you sure addon secret name used to have ```tanzu``` in it? What is an example addone secret name?

Reply:
yes: 
https://github.com/vmware-tanzu/tanzu-framework/blob/458ccb00877360826fbef34c429ca08259793195/pkg/v1/providers/ytt/02_addons/tanzu-addons-manager/add_tanzu-addons-manager.yaml#L49

Reply:
why the double Sprintf? Could we do `%s-tanzu-addons-manager-addon`

Reply:
fixed